crates/orderbook/openapi.yml

```diff
@@ -668,13 +668,24 @@ components:
                 be set to the `keccak256` hash of the UTF-8 encoded bytes of this string. You must
                 ensure that the signature that is part of this request indeed signed a smart contract
                 order with the `appData` field set appropriately. If this isn't the case, signature
-                verification will fail. For easier debugging it is recommended to additionally set the 
+                verification will fail. For easier debugging it is recommended to additionally set the
                 `appDataHash` field.
 
                 The field must be the encoding of a valid JSON object. The JSON object can contain
                 arbitrary application specific data (JSON key values). The optional key `backend` is
                 special. It **MUST** conform to the schema documented in `BackendAppData`.
 
+                The intended use of the other keys of the object is follow the standardized format
```

Comment:
What do you mean by "the other keys in the object"? 

I would love we use this opportunity to advertise how following the standard allows to attach arbitrary information to the order, this is important because it will enable among others to:
- Credit the order to a specific app
- Add additional instructions to solvers about your order preferences (i.e. CoW hooks, wait for cows, and any future protocol extension)
- Attach UTM codes to credit the order to a campaign

Reply:
"other keys" refers to the previous paragraph that introduces the `backend` key.

Reply:
I guess long term it makes sense to document all of the app data in one place. There is no reason to document BackendAppData here and the other app data somewhere else.

Reply:
Then we could even enforce that the non backend relevant app data also matches the schema.